plugins/woocommerce-blocks/assets/js/blocks/product-filters/types.ts

```diff
@@ -1,7 +1,8 @@
-/**
- * Internal dependencies
- */
-
 export interface ProductFiltersBlockAttributes {
 	productId?: string;
+	setAttributes: ( attributes: ProductFiltersBlockAttributes ) => void;
+	overlay: string;
+	overlayIcon: string;
+	overlayButtonStyle: string;
+	overlayIconSize: number | undefined;
```

Comment:
What about making this attribute optional?
```suggestion
	overlayIconSize?: number;
```